temple/utils.py

```diff
@@ -126,3 +126,30 @@ def _call_api(self, verb, url, **request_kwargs):
     def get(self, url, **request_kwargs):
         """Github API get"""
         return self._call_api('get', url, **request_kwargs)
+
+class GitLabClient():
+    def __init__(self, template):
```

Comment:
The GitHub client doesn't take the template as an argument to init; I think they should follow the same pattern.